app/contributing/style-guide.md

```diff
@@ -276,6 +276,29 @@ When deciding which icon to use for a doc, use the following guidelines:
 4. If the answer to all of the above is "no", you can
    [upload a custom image](/contributing/markdown-rules/#icons).
 
+## Documenting Third-Party Tools
+
+Our documentation sometimes requires integration with third-party tools and services to help users succeed with our products. 
+When writing docs for third-party tools and services, follow these guidelines:
+
+* **Link to third-party documentation**: Always link to the official documentation of third-party products. For example:
+  * [Add Developer Teams from Identity Providers](/konnect/dev-portal/access-and-approval/add-teams/)
+* **Include third-party instructions with integrations**: Provide instructions that involve switching between products, and include links to the official third-party documentation when possible. For example:
+  * [How to configure Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
```

Comment:
I think we need more of a preamble here with a decision process.

```suggestion
Our documentation sometimes requires integration with third-party tools and services to help users succeed with our products. 
As a general rule, we try to avoid rewriting docs that exist elsewhere. When writing docs that involve interactions with third-party tools and services, think about the following:

* Does the third-party app have a complete doc in their own documentation that covers the process?
* Does the process include a lot of custom configuration of the third-party product to get it to work with Kong? 
* Do you need to switch back and forth between Kong and the third-party to complete a task?

Depending on your answers, you should:
* **Link to third-party documentation**: Always link to the official documentation of third-party products.
In many cases, a link is enough, often as part of a step or a group of prerequisites. Here's a doc that uses this method:
  * [Add Developer Teams from Identity Providers](/konnect/dev-portal/access-and-approval/add-teams/)
  
* **Include third-party instructions with integrations**: Provide instructions that involve switching between products, and include links to the official third-party documentation when possible. For example:
  * [How to configure Transit Gateways](/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
```